library/src/rocblas_auxiliary.cpp

```diff
@@ -71,6 +72,9 @@ extern "C" rocblas_status rocblas_create_handle(rocblas_handle* handle)
     try
     {
         *handle = new _rocblas_handle();
+
+        static int dummy = (tensileInitialize(), 0);
+
```

Comment:
Please move the Tensile init before rocblas handle; Tensile being independent component need to be initialized first